api/spec/v1.yaml

```diff
@@ -267,7 +269,6 @@ paths:
       parameters:
         - *limit
         - *offset
-        - *height
```

Comment:
The `/entities`, `/accounts`, and `/validators` endpoints have this seemingly superfluous parameter that isn't read in or used from what I can tell. Maybe a vestigial parameter? Or maybe this info is somehow included in the chainID now?

Reply:
> vestigial

More like future proofing 😁